### PR TITLE
🐛 fix crash on app first startup

### PR DESCRIPTION
**Pull request by adeel0202**
On app first startup, `masterData` was null hence the app was being crashed.

### CODE REVIEW

**Review by vthunetcompany — COMMENTED** [review]
Hi @adeel0202,
I cannot reproduce this problem.
Can you provide additional information? e.g., 
- platform: iOS or Android?
- runtime: on simulators or a real device?
- version: Xcode/iOS or Android version?
- OS: Linux, macOS, or Windows? 


Nonetheless, I was thinking this could be a better solution

File: App.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const App = () => {
     loadData().then(res => {
       console.log('Startup::', res);
       if (!!res) {
-        setMasterData(res?.masterData);
+        setMasterData(res?.masterData ? res?.masterData : []);
```
Comment (vthunetcompany) [inline App.js]:
```suggestion
        setMasterData(res.masterData ?? []);
```

**Review by vthunetcompany — COMMENTED** [review]
Hi @adeel0202,
I cannot reproduce this problem.
Can you provide additional information? e.g.,
- platform: iOS or Android?
- runtime: on simulators or a real device?
- version: Xcode/iOS or Android version?
- OS: Linux, macOS, or Windows?

Nonetheless, I was thinking this could be a better solution:

**Comment by adeel0202** [comment]
If you uninstall the app from your device and then run the app again, you might see the issue, since it's only reproducible on the first startup.

Here is the screenshot:
Screenshot_1704871669


platform: Android
runtime: Pixel 4 emulator
version: Android version
OS: macOS

Also yes, your solution is better.

**Review by vthunetcompany — APPROVED** [review]
LGTM ✅

**Pull request merged**